lib/modules/manager/puppet/readme.md

```diff
@@ -0,0 +1,66 @@
+simply keeps Puppetfiles updated
+
+# supported Puppetfile formats
+
+the manager extracts the deps from one Puppetfile
+
+the Puppetfile supports at the moment different ways to configure forges
+
+1. no forge defined
+
+   ```ruby
+   mod 'puppetlabs/apt', '8.3.0'
+   mod 'puppetlabs/apache', '7.0.0'
+   ```
+
+1. one forge defined: `forge "https://forgeapi.puppetlabs.com"`
+
+   ```ruby
+   forge "https://forgeapi.puppetlabs.com"
+
+   mod 'puppetlabs/apt', '8.3.0'
+   mod 'puppetlabs/apache', '7.0.0'
+   mod 'puppetlabs/concat', '7.1.1'
+   ```
+
+1. multiple forges defined
+
+   ```ruby
+   forge "https://forgeapi.puppetlabs.com"
+
+   mod 'puppetlabs/apt', '8.3.0'
+   mod 'puppetlabs/apache', '7.0.0'
+   mod 'puppetlabs/concat', '7.1.1'
+
+   # private forge
+   forge "https://forgeapi.example.com"
+
+   mod 'example/infra', '3.3.0'
+   ```
+
+# general overview
+
+![puppet overview](__docs__/puppet_overview.dio.png)
```

Comment:
This don't work on production docs, as the image don't get published.

I don't think we need this at all, as we don't have it for other managers.

@rarkins WDYT?

Reply:
It would be nice if we could support images, but in this case I agree we don't need this diagram.